components/rpill/rust/ffi/rpill-lib/src/lib.rs

```diff
@@ -0,0 +1,38 @@
+#[allow(unused_imports)]
+use serde::Deserialize;
+#[allow(unused_imports)]
+use std::collections::HashMap;
+#[allow(unused_macros)]
```

Comment:
When writing code that targets a particular OS, a nice pattern is to make separate conditionally included modules for each platform with different behavior. That way each module is only concerned with a single platform, and you can limit all of the `cfg` attributes to a single top-level location. That way you won't need to worry about unused imports either :smile: 

Two examples in the wild of how this can be done in practice:
https://github.com/antonok-edm/vst_window/blob/master/src/platform/mod.rs#L19-L28
https://github.com/alexcrichton/filetime/blob/master/src/lib.rs#L43-L57

Reply:
Thanks for the suggestion! I'm resolving this thread as we're not relying on Rust anymore